classes/profile_table.php

```diff
@@ -146,4 +155,25 @@ public function col_user(object $record): string {
             }
         }
     }
+
+    /**
+     * Display for the delete column
+     *
+     * @param object $record
+     * @return mixed
+     * @throws \coding_exception
+     * @throws \moodle_exception
+     */
+    public function col_delete(object $record) {
+        global $OUTPUT;
+        $deleteurl = new \moodle_url('/admin/tool/excimer/delete.php', [ 'deleteid' => $record->id, 'sesskey' => sesskey() ]);
+        $link = new \action_link(
+                $deleteurl,
+                '',
+                new \confirm_action('Confirm?'),
```

Comment:
This should be a lang string (get_string call). I believe one exists for 'Confirm' under `lang/en/moodle.php`